ddl/schema.go

```diff
@@ -247,6 +247,98 @@ func onDropSchema(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 	return ver, errors.Trace(err)
 }
 
+const (
+	recoverSchemaCheckFlagNone int64 = iota
+	recoverSchemaCheckFlagEnableGC
+	recoverSchemaCheckFlagDisableGC
+)
+
+func (w *worker) onRecoverSchema(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
+	var (
+		recoverSchemaInfo      *RecoverSchemaInfo
+		recoverSchemaCheckFlag int64
+	)
+	const checkFlagIndexInJobArgs = 1 // The index of `recoverSchemaCheckFlag` in job arg list.
```

Comment:
Please change it to global level so that it could be shared by different functions(like this one and `onRecoverTable`).

Reply:
Done. PTAL ,Thx~